R/collect.R

```diff
@@ -55,6 +55,24 @@
 #' [collect_extracts()] returns a tibble with columns for the resampling
 #' indicators, the location (preprocessor, model, etc.), and objects extracted
 #' from workflows via the `extract` argument to [control functions][control_grid()].
+#'
+#' @section Extracting with submodels:
```

Comment:
This section title requires readers to already know what the submodel trick is. Can you think of a way of rephrasing that for someone just being confused with about the behavior of `collect_extacts()` and trying to figure out what's going on?

Reply:
Heard. Attempted to resolve in [54f5074](https://github.com/tidymodels/tune/pull/743/commits/54f5074432d43a8e99c78a1563f465f001dbe9bf), will leave to you to decide there.